lib/cache.js

```diff
@@ -45,30 +46,37 @@ class Cache {
     const tempFolder = path.join(os.tmpdir(), 'tldr', utils.uniqueId());
 
     // Downloading fresh copy
-    return Promise.all([
-      // Create new temporary folder
-      fs.ensureDir(tempFolder),
-      fs.ensureDir(this.cacheFolder)
-    ])
-      .then(() => {
-        // Download and extract cache data to temporary folder
-        return remote.download(tempFolder);
-      })
-      .then(() => {
-        // Copy data to cache folder
-        return fs.copy(tempFolder, this.cacheFolder);
-      })
-      .then(() => {
-        return Promise.all([
-          // Remove temporary folder
-          fs.remove(tempFolder),
-          index.rebuildPagesIndex()
-        ]);
-      })
-      // eslint-disable-next-line no-unused-vars
-      .then(([_, shortIndex]) => {
-        return shortIndex;
-      });
+    return (
```

Comment:
Do we need to reform this entire block of code, or can the previous indentation be left in place? Personally seeing a `return (` just feels weird vs having `return Promise.all([` on the one line.

Reply:
Looks like this got missed. I went ahead and made this change in 04a8cf2414edb0c82ef02f29a355c9f4b27dcccf.